test/node/test_entrance.rb

```diff
@@ -60,6 +60,7 @@ def test_pushes_wallet
   end
 
   def test_renders_json
+    skip
```

Comment:
@emilianodellacasa why?

Reply:
@emilianodellacasa you did it wrong. When you skip a test, you should add a comment in the code for it (a puzzle). We can't merge the `skip` without any additional text for it. In a few months someone will look at this code and ask: WTF? Where he/she will find an answer?